app/validations/site-validation.js

```diff
@@ -1,7 +1,8 @@
 import Joi from 'joi';
 import { CLASSIFICATION_CODE } from 'frontend-contactgegevens-loket/models/administrative-unit-classification-code';
 import { SITE_CODE } from '../models/site';
-const belgiumPhoneNumberRegex = /^(tel:)?(\+32|04|0032)[1-9](?:\s?[0-9]){7,8}$/;
+const belgiumPhoneNumberRegex =
+  /^(tel:)?(\+32|04|0032|03)[1-6](?:\s?[0-9]){6,9}$/;
```

Comment:
the 03 for a belgium phone number like 032331122 because it detected it as a non belgium phone number

Reply:
Okay I see, in that case the proposed fix only covers part of the reported problem. For example, I still get the message concerning non-belgium phone numbers for the official phone numbers of several municipalities (numbers copied from their websites):
- `09 210 10 10` for Ghent
- `011 23 95 40` for Hasselt
- `016 27 27 27` for Leuven
- `03 22 11 333` for Antwerp

I *think* the regex needs some revision, from the top of my head I think it should try to match the following parts:
- optional(?) country code (`0032` or `+32`)
- zone number, with or without leading zero depending on whether a country code was present OR `[0]4` to deal with mobile phone numbers (`04` is also the zone number for Liege so that might become tricky)
- remaining digits (check for correct amount, which might depend the previous parts)

Reply:
What is your suggestion for this problem ?

Reply:
Ideally we can define a regex (or some other validation method) that correctly matches all Belgian phone numbers. But I doubt this is going to be trivial to get right. I would raise this topic in bug ticket so that it can be discussed in terms priority and complexity against other tickets, and that we might concretely define how strict this validation should be.